test/utils.js

```diff
@@ -121,6 +125,28 @@ const utils = {
   },
 
   getReceiptValue: (receipt, arg) => receipt.logs[0].args[arg],
+
+  proposeReparamAndGetPropID: async (reParam, value, actor) => {
+    const parameterizer = await Parameterizer.deployed();
+    const receipt = await utils.as(actor, parameterizer.proposeReparameterization, reParam, value);
+    return receipt.logs[0].args.propID;
+  },
+
+  challengeReparamAndGetChallengeID: async (propID, actor) => {
+    const parameterizer = await Parameterizer.deployed();
+    const receipt = await utils.as(actor, parameterizer.challengeReparameterization, propID);
+    return receipt.logs[0].args.pollID;
+  },
+
+  decimalDivide: (numerator, denominator) => {
```

Comment:
`decimalDivideEther`

Reply:
I actually don't totally understand how you're using this function. For example you have:
```
utils.decimalDivide(
  paramConfig.pDispensationPct, 100,
)
```
So pDispensationPct is getting `toWei`d which seems... hacky? Because pDispensationPct is not a unit in Ether.

Reply:
Hi @skmgoldin , please take a look at my new updated utility function. It has much better semantics for each purpose.

> So pDispensationPct is getting toWeid which seems... hacky?

Yeah, it feels somewhat hacky, but it's what ethjs recommends: https://github.com/ethjs/ethjs/blob/master/docs/user-guide.md#big-numbersnumber-handling